llvm/test/Transforms/InstCombine/2006-10-26-VectorReassoc.ll

```diff
@@ -4,9 +4,8 @@
 ; Verify this doesn't fold when no fast-math-flags are specified
```

Comment:
About that comment, I think maybe there is a bug in the LICM pass specifically in the hoistMulAddAssociation(..) or hoistBOAssociation(), because the idea of folding with fast-math-flags only is not related to the associativity of FMul.